### PR TITLE
Loot tables to SQL DB

### PR DESCRIPTION
**Pull request by grimmier378**
Migrated the loot tables to an SQL DB with tables for (Global_Items,  Normal_Items)

This allows the data to be shared by all of the characters without to many conflicts.

If you edit or add anything on the driver and save the settings you can issue /rgl lootreload on all characters to reload from the DB. 

## IMPORTANT !!

The **FIRST TIME** you run this after this update, **ONLY** Load RGMercs Lua On **ONE Character** to build the DB. 

This Could take a couple seconds to create and populate the database. (_especially if your loot.ini is large_)

After you have the database created you can load as many toons as you would like. 

## Loading Order of Operation.

* Check for loot DB (saved by server name cause emu servers can have different loot)
  * If the DB is missing we create it and the 2 tables.
  * If we created a new Database
    * then load working tables from  loot.ini file and write their contents to the database tables.
  * If we already have the DB then load our working tables from that.
  * Pass the Working tables back to the Loot Module

## Notes:

Any item changes and new additions are written to the ini file for non mercs users, and to the Database. Buy items are per character so we will keep those in our own tables since we do not need to share them.

### CODE REVIEW

**Comment by grimmier378** [comment]
ok ready for your review there Derple

File: lib/lootnscoot/loot_lib.lua [hunk 1]
```diff
@@ -110,17 +110,18 @@ There is also no flag for combat looting. It will only loot if no mobs are withi
...
 
-local mq = require 'mq'
+local mq           = require 'mq'
 
-local eqServer = string.gsub(mq.TLO.EverQuest.Server(), ' ', '_')
+local eqServer     = string.gsub(mq.TLO.EverQuest.Server(), ' ', '_')
 -- Check for looted module, if found use that. else fall back on our copy, which may be outdated.
-local eqChar = mq.TLO.Me.Name()
-local actors = require('actors')
+local eqChar       = mq.TLO.Me.Name()
```
Comment (DerpleMQ2) [inline lib/lootnscoot/loot_lib.lua:117]: config globals already has your name./

File: lib/lootnscoot/loot_lib.lua [hunk 2]
```diff
@@ -110,17 +110,18 @@ There is also no flag for combat looting. It will only loot if no mobs are withi
...
+local RGMercUtils  = require("utils.rgmercs_utils")
+local actors       = require('actors')
```
Comment (DerpleMQ2) [inline lib/lootnscoot/loot_lib.lua:119]: RGMercUtils already imports and makes available actors don't really need to import it again.

**Review by DerpleMQ2 — CHANGES_REQUESTED** [review]

File: lib/lootnscoot/loot_lib.lua (hunk 1 above)
Comment (grimmier378) [reply lib/lootnscoot/loot_lib.lua:117]: another remnant will remove

File: lib/lootnscoot/loot_lib.lua (hunk 2 above)
Comment (grimmier378) [reply lib/lootnscoot/loot_lib.lua:119]: ahh good call

**Pull request merged**